dask/array/tests/test_cupy.py

```diff
@@ -1468,3 +1468,120 @@ def test_cupy_cholesky(shape, chunk):
         check_graph=False,
         check_chunks=False,
     )
+
+
```

Comment:
Can you also add the `@pytest.mark.gpu` line.  xref https://github.com/dask/dask/pull/7876 for more details

Reply:
I don't know for sure, but it seems like https://github.com/dask/dask/blob/main/dask/array/tests/test_cupy.py#L5 is marking the whole file as GPU. No single tests have that mark in that file. Could you confirm @charlesbluca ?

Reply:
Yup, this entire file is marked for GPU testing

Reply:
Ah!  Sorry for the noise then